### PR TITLE
SAK-50456 Sitestats improve ui responsiveness

### PR DESCRIPTION
**Pull request by Sharadhi98**
There was an issue with the "Activity" widget being left-aligned while the rest were center-aligned which is now fixed.

### CODE REVIEW

File: library/src/skins/default/src/sass/modules/tool/sitestats/_sitestats.scss [hunk 1]
```diff
@@ -134,12 +134,17 @@
...
     margin-left: 100px;
+    border-left: 1px solid var(--sakai-border-color);
   }
 
   .widget .ministat {
     padding: 0 10px;
     border-left: 1px solid var(--sakai-border-color);
     text-align: center;
+	@media #{$phone}{
+	padding: 0 60px;
```
Comment (jesusmmp) [inline library/src/skins/default/src/sass/modules/tool/sitestats/_sitestats.scss:145]: Spacing

Comment (adrianfish) [inline library/src/skins/default/src/sass/modules/tool/sitestats/_sitestats.scss:145]:
```suggestion
    @media #{$phone} {
      padding: 0 60px;
```

Comment (adrianfish) [reply library/src/skins/default/src/sass/modules/tool/sitestats/_sitestats.scss:145]: @Sharadhi98 If you could just fix the indentation on these lines, we can get this merged. We try to use spaces in our files where we can, and this file seems to be using 2 spaces.

**Review by adrianfish — CHANGES_REQUESTED** [review]

File: library/src/skins/default/src/sass/modules/tool/sitestats/_sitestats.scss (hunk 1 above)
Comment (Sharadhi98) [reply library/src/skins/default/src/sass/modules/tool/sitestats/_sitestats.scss]: Thank you, I pushed the updated code.

**Review by adrianfish — APPROVED** [review]
Thanks Sharadhi.

**Pull request merged**